apps/avifdec.c

```diff
@@ -387,5 +388,6 @@ int main(int argc, char * argv[])
     if (decoder != NULL) {
         avifDecoderDestroy(decoder);
     }
+    FREE_ARGV()
```

Comment:
IMPORTANT: At line 385, I think we also need to check `decoder` now, because we may goto cleanup from inside `NEXT_ARGV()` before `decoder` is allocated.

We can probably just move lines 385-387 right after line 388.

Reply:
Right, it is set to NULL at first which does not trigger the destroy line 389.